schematic/models/metadata.py

```diff
@@ -235,3 +237,45 @@ def populateModelManifest(self, title, manifestPath: str, rootNode: str) -> str:
         emptyManifestURL = mg.get_manifest()
 
         return mg.populate_manifest_spreadsheet(manifestPath, emptyManifestURL)
+
+
+    def submit_metadata_manifest(self, manifest_path: str, dataset_id: str, validate_component: str, json_schema: str = None) -> bool:
```

Comment:
if validate_component is None and json_schema is None, then just store metadata manifest to the dataset folder.

Reply:
In the case-scenario where `validate_component` and `json_schema` both are `None`, the manifest file is just pushed to the dataset folder. Handler on [L#305](https://github.com/Sage-Bionetworks/schematic/blob/develop-submit_metadata_manifest/schematic/models/metadata.py#L305).